### PR TITLE
Feature histogram

### PR DESCRIPTION
**Pull request by robinschmid**
finalization of #1896

### CODE REVIEW

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+/**
+ * A histogram for features of one or multiple feature lists
+ */
+public class FeatureHistogramPlotController extends
+    FxController<FeatureHistogramPlotModel> implements SelectedFeatureListsBinding {
+
+  private static final Logger logger = Logger.getLogger(
+      FeatureHistogramPlotController.class.getName());
+  private static final org.slf4j.Logger log = LoggerFactory.getLogger(
```
Comment (SteffenHeu) [inline mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java:55]: I remember someone git shamed me already for this :D

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  }
+
+  private void initializeListeners() {
+    logger.info("Initializing feature histogram plot listeners.");
+    PropertyUtils.onChange(this::computeDataset, model.featureListsProperty(),
+        model.selectedTypeProperty());
+
+    // first listen for changes to typeChoices
+    model.getTypeChoices().addListener((ListChangeListener<NumberType>) _ -> {
+      logger.info(() -> "Type choices changed");
```
Comment (SteffenHeu) [inline mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java:81]: is this worth info level?

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotViewBuilder.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  public FeatureHistogramPlotViewBuilder(FeatureHistogramPlotModel model) {
+    super(model);
+  }
+
+  @Override
+  public Region build() {
+    final HistogramPanel histo = new HistogramPanel();
+    final BorderPane mainPane = new BorderPane(histo);
+    mainPane.setTop(createDataTypeBox());
```
Comment (SteffenHeu) [inline mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotViewBuilder.java:49]: usually we put the controls below the plot, e.g. spectra plot, overview, stats...

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramDatasetProvider.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import io.github.mzmine.gui.chartbasics.simplechart.providers.SimpleXYProvider;
+import java.awt.Color;
+import java.text.DecimalFormat;
+import org.jetbrains.annotations.Nullable;
+
+public class FeatureHistogramDatasetProvider extends SimpleXYProvider {
+
+  private final NumberType dataType;
+
+  public FeatureHistogramDatasetProvider(Color color, String key, NumberType dataType) {
```
Comment (SteffenHeu) [inline mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramDatasetProvider.java:38]: This class is not used or?

**Review by SteffenHeu — COMMENTED** [review]
Some small comments, looks good though

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramDatasetProvider.java (hunk 4 above)
Comment (robinschmid) [reply mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramDatasetProvider.java:38]: true. Olena musst have copied it

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java (hunk 1 above)
Comment (robinschmid) [reply mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java:55]: maybe it was copied from another class not sure how this got here

File: mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java (hunk 2 above)
Comment (robinschmid) [reply mzmine-community/src/main/java/io/github/mzmine/modules/visualization/feat_histogram/FeatureHistogramPlotController.java:81]: changed to fine

**Comment by robinschmid** [comment]
@SteffenHeu ready from my side

**Pull request merged**